### PR TITLE
Corrected script for some item #26

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

File: sql-files/mob_db.sql [hunk 1]
```diff
@@ -62,6 +62,7 @@ CREATE TABLE `mob_db` (
   `racegroup_ep172bath` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_illusion_turtle` tinyint(1) unsigned DEFAULT NULL,
   `racegroup_rachel_sanctuary` tinyint(1) unsigned DEFAULT NULL,
+  `racegroup_Illusion_Luanda` tinyint(1) unsigned DEFAULT NULL,
```
Comment (Pokye) [inline sql-files/mob_db.sql:65]:
```suggestion
  `racegroup_illusion_luanda` tinyint(1) unsigned DEFAULT NULL,
```
Isn't it better to follow the standard and leave all "racegroup_Illusion_Luanda" with lowercase letters?

Comment (idk-whoami) [reply sql-files/mob_db.sql]: Thanks!

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -177855,6 +183777,30 @@ Body:
...
+    View: 4
+    Script: |
+      .@r = getrefine();
+      bonus bUnbreakableShield;
+      bonus2 bSubClass,Class_Boss,40;
+      bonus2 bSubClass,Class_Normal,40;
+      bonus bMaxHPrate,3*.@r;
+      bonus bMaxSPrate,3*.@r;
+      if (.@r>=5)
+         bonus2 bSubEle,Ele_Holy,25-(getskilllv("CR_TRUST")*25);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:183801]: Are you sure about `getskilllv("CR_TRUST")*25` ?
Maybe I read it wrong, for me it should be 
```suggestion
         bonus2 bSubEle,Ele_Holy,25-(getskilllv("CR_TRUST") > 0 ? 25 : 0);
```

Comment (idk-whoami) [reply db/re/item_db_equip.yml:183801]: Thanks!

**Review by Atemo — APPROVED** [review]
Any plan to implement the missing bonus commands ?
- `When Equipped Adds Earth Attribute to Bullets for 180 Minutes`
- `When hit by a physical attack There is a chance that player will die - Refine will reduce the chance`

**Comment by idk-whoami** [comment]
no
i have no idea and knowledge to do this
T_T

**Comment by mazvi** [comment]
please check this item https://www.divine-pride.net/database/item/620010/illusion-doom-slayer on rathena this item only blacksmith class?

**Pull request merged**